documentation/examples.md

```diff
@@ -92,28 +92,26 @@ D('example.com', REG, DnsProvider('R53'),
 )
 ```
 
-## Add comments along complex SPF records
-
-You normally can't put comments in the middle of a string,
-but with a little bit of creativity you can document
-each element of an SPF record this way.
+## Use SPF_BUILDER to add comments along SPF records
 
 ```javascript
-var SPF_RECORDS = TXT('@', [
-    'v=spf1',
-    'ip4:1.2.3.0/24',           // NY mail server
-    'ip4:4.3.2.0/24',           // CO mail server
-    'include:_spf.google.com',  // Google Apps
-    'include:mailgun.org',      // Mailgun (requested by Ticket#12345)
-    'include:servers.mcsv.net', // MailChimp (requested by Ticket#54321)
-    'include:sendgrid.net',     // SendGrid (requested by Ticket#23456)
-    'include:spf.mtasv.net',    // Desk.com (needed by IT team)
-    '~all'
-].join(' '));
-
-D('example.com', REG, DnsProvider('R53'),
-   SPF_RECORDS,
-)
+D("example.tld", REG, DSP, ...
+  A("@", "10.2.2.2"),
+  MX("@", "example.tld."),
+  SPF_BUILDER({
+    label: "@",
+    overflow: "_spf%d",
+    raw: "_rawspf",
```

Comment:
To emphasize the commenting, I'd remove any non-default fields and add some more `parts`.

Suggestion:
* Remove:  `overflow`, `raw`, `ttl`
* Add some items that would be confusing without a comment:

```
      "include:mailgun.org", // Greenhouse.io
      "include:servers.mcsv.net", // MailChimp
      "include:450622.spf05.hubspotemail.net", // Hubspot (Ticket#12345)
```